### DISCUSSION TITLE
How can I install Podman in HPC cluster

### DISCUSSION THREAD

**Discussion by Zihe33Huang** [containers/podman#18225] [General]
### Feature request description

I want to do experiments on an HPC cluster, but I can't install Podman without sudo, is there any good way to solve this problem?

**Comment by grantcurell** [comment]
I am also looking for a way to do this.

The problem is the admin seems to have removed /etc/subuid and /etc/subgid and I don't think there is a way to get around that without asking the admin to change it.

@Zihe33Huang - as far as Podman goes, you could compile it from source locally but even that assumes you have the right packages.

In total, it seems like the initial install is going to have to be done by an admin.

**Reply by Zihe33Huang** [reply]
谢谢你! 看了你的主页，你会说中文，所以我用中文来表达我的感谢😊

**Reply by Zihe33Huang** [reply]
hi @grantcurell , now I have found a way to install Podman on HPC system and I wrote a note, hope this would be helpful.  https://github.com/Zihe33Huang/HPC_Containers/wiki/How-to-install-Podman-on-HPC-system

**Reply by afbjorklund** [reply]
You should use fuse-overlayfs if at all possible, since the performance and disk usage of "vfs" is horrible.

**Reply by Zihe33Huang** [reply]
> You should use fuse-overlayfs if at all possible, since the performance and disk usage of "vfs" is horrible.

Thank you!  I will try it!

**Comment by afbjorklund** [comment]
You can install it by compiling everything, including dependencies, from source. Using packages is optional, although much more convenient...

But maintaining the subuid tables requires admin support, otherwise you are stuck with one user per container (and "ignore_chown_errors")
